packages/localize-tools/testdata/build-transform-xliff/input/xliff/es-419.xlf

```diff
@@ -46,6 +46,18 @@
   <source>&lt;Hello<ph id="0">&lt;b></ph>&lt;World &amp; Friends><ph id="1">&lt;/b></ph>!></source>
   <target>&lt;Hola<ph id="0">&lt;b></ph>&lt;Mundo &amp; Amigos><ph id="1">&lt;/b></ph>!></target>
 </trans-unit>
+<trans-unit id="s63f0bfacf2c00f6b">
+  <source>Hello</source>
+  <target>Hola</target>
+</trans-unit>
+<trans-unit id="s372f95c2b25986c8">
+  <source>Different <ph id="0">${user}</ph></source>
```

Comment:
`<source>` expression is generated by the `extract` command but never read again by lit-localize so it does not need to really match anything. It will be overwritten if `extract` command is run again and finds a new value from the source code.